src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -24,6 +31,7 @@
 import java.lang.reflect.Field;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.net.HttpURLConnection;
```

Comment:
Sorry, the rest of the work got lost in the diff.
This provides enums for HTTP status codes, so instead of comparing a code to, e.g., `404`, we compare it to `HttpUrlConnection.NOT_FOUND`
Example: https://github.com/EasyPost/easypost-java/pull/80/files#diff-562d3c98a6e71dc50980fe8300ab00742293eb925eeb5f106f0086704030519aR392

Reply:
Ah I see, thanks for clarifying.